tests/python_client/testcases/test_search.py

```diff
@@ -428,6 +429,29 @@ def test_search_param_invalid_expr_bool(self, get_invalid_expr_bool_value):
                             check_items={"err_code": 1,
                                          "err_msg": "failed to create query plan"})
 
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_search_with_expression_invalid_like(self, dim, _async):
```

Comment:
Here "dim" is not defined in invalid class.

Reply:
And check "_async" too.

Reply:
> Here "dim" is not defined in invalid class.

I get it. And I'll make a change.

Reply:
> And check "_async" too.

I think "_async" is not needed in invalid case, so i delete it.